lib/git_reflow/git_helpers.rb

```diff
@@ -40,11 +40,12 @@ def fetch_destination(destination_branch)
       run_command_with_label "git fetch origin #{destination_branch}"
     end
 
-    def update_destination(destination_branch)
+    def update_destination(destination_branch, remote = 'origin')
```

Comment:
I would still like to see the `remote = 'origin'` argument for this method change to an `options = {}`